src/main/resources/AdditionalLogin.fxml

```diff
@@ -20,5 +20,6 @@
             <Button fx:id="loginButton" layoutX="69.0" layoutY="161.0" mnemonicParsing="false" onAction="#loginButtonClicked" prefHeight="25.0" prefWidth="93.0" text="Log-In" />
          </children>
       </Pane>
+      <Label fx:id="i_label" layoutX="416.0" layoutY="200.0" text="INCORRECT!" textFill="RED" visible="false" />
```

Comment:
we should be following those ideas in the slides on UI design. I would not consider this polite.

Reply:
also we want to distinguish between incorrect username and password.